pages/scenario_selection_page.py

```diff
@@ -279,14 +289,17 @@ def scenario_selection_page():
                         == "rebalance unit costs between electricity and gas"
                     ):
                         with levies_inputs_col:
-                            variable_electricity_weight = st.slider(
-                                label="Rebalance between electricity (0) <-> gas (100)",
-                                min_value=0.0,
-                                max_value=100.0,
-                                value=0.0,
-                                step=10.0,
-                                key="custom_levy_variable_electricity_weights_input",
-                                help="Proportion of the scheme revenue that is levied against electricity units",
+                            custom_gas_weight = (
+                                st.slider(
+                                    label="Rebalance between electricity (0) <-> gas (100)",
+                                    min_value=0.0,
+                                    max_value=100.0,
+                                    value=0.0,
+                                    step=10.0,
+                                    key="custom_levy_gas_weights_input",
+                                    help="Proportion of the scheme revenue that is levied against gas units",
```

Comment:
```suggestion
                                    help="Proportion of the scheme revenue that is levied against gas units. Electricity (0) setting is 0% of scheme revenue is levied on gas units (i.e. 100% on electricity) and gas (100) setting is 100% of scheme revenue on gas units.",
```

Implementing suggested change from feedback slide in sprint 4 deck